### PR TITLE
Python

### LINKED ISSUE #574: Python: Advanced functions-lambda
### Description :
What is `lambda` in python?

### Note : 

- Changes should be made inside the `Python/` directory & `Python` branch.
-  Issue only for `GWOC'21` contributors.
-  Issue will be assigned on a **first come first serve basis**, **1 Issue  ==  1 PR.**  
-  In the PR, keep this issue's title as your PR's title.

#### Must Follow : [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of Conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) before start Contributing.

### PR DESCRIPTION
**Pull request by srini047**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #574 

<hr>

## Type of change 📝

- [ ] Audio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [x] Video (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [x] Python_Domain
<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by srini047** [comment]
Please review my PR @prathimacode-hub.

**Review by kumarankm — APPROVED** [review]
Looks fine, good job 👍🏻 
You can merge this pr @adityabisoi

File: Python/Advanced_Function_Lambda/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+[![Advanced Python Functions: Lambda (V)](Lambda-574-Thumbnail.png)](https://drive.google.com/file/d/10jVjCcxN67BbPKQPOxnIyuyTh_D7LT8Y/view?usp=sharing "Lambda Functions (Video)")
```
Comment (apeksha235) [inline Python/Advanced_Function_Lambda/README.md:1]: @srini047 amazing work on the video! just make your md file more presentable: 
- Give a proper heading 
- Write down the subtopics covered in the video

**Review by apeksha235 — CHANGES_REQUESTED** [review]

File: Python/Advanced_Function_Lambda/README.md (hunk 1 above)
Comment (srini047) [reply Python/Advanced_Function_Lambda/README.md:1]: I have changed the READM.md file as per the suggestions above. @apeksha235

Comment (apeksha235) [reply Python/Advanced_Function_Lambda/README.md:1]: Amazing! @adityabisoi please merge this

**Pull request merged**